packages/ui/src/components/modals/ChangeMultisig.tsx

```diff
@@ -342,10 +351,22 @@ const ChangeMultisig = ({ onClose, className }: Props) => {
       fullWidth
       maxWidth={'sm'}
       open
-      onClose={handleClose}
       className={className}
     >
-      <DialogTitle>Change multisig</DialogTitle>
+      <DialogTitle>
+        Change multisig
+        {onClose ? (
+          <IconButton
+            className="closeButton"
+            size="small"
+            aria-label="close"
+            color="inherit"
+            onClick={onClose}
+          >
+            <CloseIcon fontSize="small" />
+          </IconButton>
+        ) : null}
+      </DialogTitle>
```

Comment:
I suggest to place it as the first element, outside of the `DialogTitle` and also no need to check for the `onClose`, this will never be undefined.

```suggestion
        <IconButton
          className="closeButton"
          size="small"
          aria-label="close"
          color="inherit"
          onClick={onClose}
        >
          <CloseIcon fontSize="small" />
        </IconButton>
      <DialogTitle>Change multisig</DialogTitle>
```

Reply:
Done, Thanks!!